src/main/java/org/jabref/gui/copyfiles/CopyFilesDialogController.java

```diff
@@ -0,0 +1,58 @@
+package org.jabref.gui.copyfiles;
+
+import javax.inject.Inject;
+
+import javafx.event.ActionEvent;
+import javafx.fxml.FXML;
+import javafx.scene.control.TableColumn;
+import javafx.scene.control.TableView;
+import javafx.scene.paint.Color;
+import javafx.scene.text.Text;
+
+import org.jabref.gui.AbstractController;
+import org.jabref.gui.util.ValueTableCellFactory;
+
+import de.jensd.fx.glyphs.materialdesignicons.MaterialDesignIcon;
+import de.jensd.fx.glyphs.materialdesignicons.utils.MaterialDesignIconFactory;
+
+public class CopyFilesDialogController extends AbstractController<CopyFilesDialogViewModel> {
+
+    @FXML private TableView<CopyFilesResultItemViewModel> tvResult;
+    @FXML private TableColumn<CopyFilesResultItemViewModel, MaterialDesignIcon> colStatus;
+    @FXML private TableColumn<CopyFilesResultItemViewModel, String> colMessage;
+    @FXML private TableColumn<CopyFilesResultItemViewModel, String> colFile;
+
+    @Inject private CopyFilesResultListDependency copyfilesresultlistDependency; //This var must have the same name as the key in the View
+
+    @FXML
+    private void close(@SuppressWarnings("unused") ActionEvent event) {
+        getStage().close();
+    }
+
+    @FXML
+    private void initialize() {
+        viewModel = new CopyFilesDialogViewModel(copyfilesresultlistDependency);
+        setupTable();
+    }
+
+    private void setupTable() {
+        colFile.setCellValueFactory(cellData -> cellData.getValue().getFile());
+        colMessage.setCellValueFactory(cellData -> cellData.getValue().getMessage());
+        colStatus.setCellValueFactory(cellData -> cellData.getValue().getIcon());
+
+        colFile.setCellFactory(new ValueTableCellFactory<CopyFilesResultItemViewModel, String>().withText(item -> item).withTooltip(item -> item));
```

Comment:
is there a way to prevent that fourth column appearing https://github.com/JabRef/jabref/pull/3147#issuecomment-339389635 ?